atlasdb-dbkvs/src/main/java/com/palantir/atlasdb/keyvalue/dbkvs/impl/oracle/OracleOverflowWriteTable.java

```diff
@@ -216,6 +219,60 @@ public void delete(List<Entry<Cell, Long>> entries) {
                 args);
     }
 
+    @Override
+    public void delete(RangeRequest range) {
+        String shortTableName = getShortTableName();
+
+        switch (config.overflowMigrationState()) {
```

Comment:
That would be nice.
Also in general, where an enum that's often getting `switch`ed over, there's scope for pushing the switch statements into classes.